sdk/src/main/java/io/dapr/client/DaprClient.java

```diff
@@ -559,6 +562,39 @@ Mono<Void> executeStateTransaction(String storeName,
    */
   Mono<Map<String, Map<String, String>>> getBulkSecret(GetBulkSecretRequest request);
 
+  /**
+   * Tries to get a lock with an expiry.
+   * @param storeName Name of the store
+   * @param resourceId Lock key
+   * @param lockOwner The identifier of lock owner
+   * @param expiryInSeconds The time before expiry
+   * @return Whether the lock is successful
+   */
+  Mono<Boolean> tryLock(String storeName, String resourceId, String lockOwner, Integer expiryInSeconds);
```

Comment:
Move all the new method definitions into the Preview Client interface since the distributed lock api is still in alpha. Only stable api methods go in this DaprClient interface....

Reply:
My mistake, I will adjust this.